### PR TITLE
update faq

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@rudygalfi PTAL

File: _support/faqs.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Speed matters and instant is the ideal. [Research](https://www.soasta.com/blog/m
...
+In addition, AMP files can be cached in the cloud in order to reduce the time content takes to get a user’s mobile device. By using the AMP format, content producers are making the content in AMP files available to be ached by third parties. Under this type of framework, publishers continue to control their content, but platforms can easily cache or mirror the content for optimal delivery speed to users. Google has stated that it will provide a cache that can be used by anyone at no cost, and all AMPs will be cached by Google’s cache. Other companies may build their own cache as well.
```
Comment (rudygalfi) [inline _support/faqs.md]: missing "c": available to be cached

**Comment by rudygalfi** [comment]
One missing letter, but LGTM

**Comment by rudygalfi** [comment]
Please just fix the missing letter and merge.

File: _support/faqs.md (hunk 1 above)
Comment (erwinmombay) [inline _support/faqs.md]: done.

**Pull request merged**